tests/unit/test_reddit.py

```diff
@@ -54,3 +54,6 @@ def test_submission(self):
 
     def test_subreddit(self):
         assert self.reddit.subreddit('redditdev').display_name == 'redditdev'
+
+    def test_comment(self):
+        assert self.reddit.comment('cklfmye').id == 'cklfmye'
```

Comment:
:space_invader: When adding to the end of the file (shouldn't be an issue once in sorted order) please try to ensure there is a trailing new line.